.github/workflows/build.yml

```diff
@@ -45,15 +48,41 @@ jobs:
        run: |
          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
          ./cmp_tags.sh
+
+     - name: Cache BuildX (BuildKit)
+       uses: actions/cache@v2
+       with:
+         path: /tmp/.buildx-cache
+         # Key should be unique for each run because if not we get an "exact cache hit" that will result not saving cache
```

Comment:
But it will know how to load the cache from non-exact keys?

Reply:
not relevant, resolving